ml-pipeline/content.md

```diff
@@ -0,0 +1,344 @@
+# ML Pipeline
+
+
+## Install Anaconda on your computer
+
+Download and install Anaconda. Select the Python 3.* version):
+https://www.anaconda.com/download/
+
+When Anaconda is installed, open "Anaconda Prompt" or any other terminal where you have ```conda``` available now.
+
+## Make an isolated Python environment
+Run ```conda create --name ml-pipeline python=3.5``` in your terminal.
+Then, to activate your new environment, run ```conda activate ml-pipeline```.
+
+
+##  Install the required packages
+
+Run ```pip install mlflow[extras]==1.1.0 mflux-ai kedro keras tensorflow```  in your terminal.
```

Comment:
Specify a range of versions of kedro (I would prefer to pin an exact version here, since Kedro is young and not stable yet), keras, tensorflow (e.g. is it tensorflow version 1 or 2?)